easybuild/easyconfigs/n/NAG/NAG-26-GCC-6.4.0-2.28.eb

```diff
@@ -0,0 +1,40 @@
+easyblock = 'Binary'
+
+name = 'NAG'
+version = '26'
+
+homepage = 'http://www.nag.co.uk'
+description = """The worlds largest collection of robust, documented, tested and maintained numerical algorithms."""
+
+toolchain = {'name': 'GCC', 'version': '6.4.0-2.28'}
+
+nagname = 'fll6i%sdfl' % version
+
+source_urls = ['https://www.nag.co.uk/downloads/impl']
+sources = [nagname + '.tgz']
```

Comment:
No need to use the `nagname` local variable if you're going to use `PackedBinary`. Also, please use the `%(version)s` template:

```python
sources = ['fll6i%(version)sdfl.tgz']